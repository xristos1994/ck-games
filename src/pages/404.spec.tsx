import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './404';

describe('pages/404', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <NotFound />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
