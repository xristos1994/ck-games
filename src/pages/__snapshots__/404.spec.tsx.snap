// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/404 renders correctly 1`] = `"Page Not Found"`;

exports[`pages/404 renders correctly 2`] = `"Page Not Found"`;
